doc/source/theory/FlavorSpace.rst

```diff
@@ -75,30 +76,125 @@ The mapping between the Evolution Basis and the +/- Basis is given by
+  \mathcal F \sim  \mathcal F_{iev,3} &= \span(\gamma, g, \Sigma_{(3)}, V_{(3)}, V_3, V_8, T_3, T_8, c^+, c^-, b^+, b^-, t^+, t^-)
 
-where we defined :math:`\Sigma_{(3)} = \sum\limits_{j=1}^3 q_j^+` and :math:`V_{(3)} = \sum\limits_{j=1}^3 q_j^-`
-(not to be confused with the usual :math:`V_3`).
+where :math:`V_{(3)}` is to be confused with the usual (QCD like) :math:`V_3`.
 
-- for :math:`n_f=6` the Intrinsic Evolution Basis coincides with the Evolution Basis: :math:`\mathcal F_{iev,6} = \mathcal F_{ev}`
+- for :math:`n_f=6` the Intrinsic QCD Evolution Basis coincides with the QCD Evolution Basis: :math:`\mathcal F_{iev,6} = \mathcal F_{ev}`
 - this basis is *not* normalized with respect to the canonical Flavor Basis
 - the basis transformation from the Flavor Basis is implemented in
   :meth:`~eko.evolution_operator.flavors.pids_from_intrinsic_evol`
 - note that for the case of non-intrinsic component the higher elements in :math:`\mathcal F_{ev}` do become linear dependent
   to other basis vectors (e.g. :math:`\left. T_{15}\right|_{c^+ = 0} = \Sigma`) but are non zero - instead in :math:`\mathcal F_{iev,3}`
   this direction vanishes
+- the photon is just a spectator and does not couple to anyone
+
+
+Unified Evolution Basis
+-----------------------
+
+In presence of QED corrections to DGLAP evolution equations,
+the QCD Evolution basis does not decouple the distributions
+as it was for the pure QCD evolution.
+
+Defining the following combinations
+
+.. math ::
+  \Sigma_u & = u^+ + c^+ + t^+ \\
+  \Sigma_d & = d^+ + s^+ + b^+ \\
+  V_u & = u^- + c^- + t^- \\
+  V_d & = d^- + s^- + b^- \\
+
+we have that in this case the QED :math:`\otimes` QCD evolution basis that performs the maximal decoupling is given by:
+
+.. math ::
+  \Sigma &= \Sigma_u + \Sigma_d \\
+  \Sigma_{\Delta} &= \Sigma_u - \Sigma_d \\
+  V &= V_u + V_d \\
+  V_{\Delta} &= V_u - V_d \\
+  T_3^u &=u^+ - c^+ \\
+  T_8^u &=u^+ + c^+ - 2t^+ \\
+  T_3^d &=d^+ - s^+ \\
+  T_8^d &=d^+ + s^+ - 2b^+ \\
+  V_3^u &=u^- - c^- \\
+  V_8^u &=u^- + c^- - 2t^- \\
+  V_3^d &=d^- - s^- \\
+  V_8^d &=d^- + s^- - 2b^- \\
+  \mathcal F \sim  \mathcal F_{QED\otimes{}QCD} &= \span(\gamma, g, \Sigma, \Sigma__{\Delta}, V, V__{\Delta}, T_3^u, T_8^u, T_3^d, T_8^d, V_3^u, V_8^u, V_3^d, V_8^d)
+
+
+- this basis is *not* normalized with respect to the canonical Flavor Basis
+- The singlet :math:`\Sigma` is just the QCD singlet
+- The valence :math:`V` is just the QCD valence
+
+
+Intrinsic Unified Evolution Basis
+---------------------------------
+
+Again, we need the generalization to the presence of intrinsic (static) distributions.
+As QED can distinguish between up-like and down-like flavors the situation is again slightly
+more involved.
+
+For :math:`n_f=3` light flavors we find:
+
+.. math ::
+  \Sigma_{(3)} &= u^+ + d^+ + s^+\\
+  T_{0,(3)} &= 2u^+ - d^+ - s^+ \\
+  V_{(3)} &= u^- + d^- + s^-\\
+  V_{0,(3)} &= 2u^- - d^- - s^-\\
+  T_1^d &=d^+ - s^+ \\
+  V_1^d &=d^- - s^- \\
```

Comment:
This is still the old version, inconsistent with the current `T_3^d`, `V_3^d`, ...